tsconfig.json

```diff
@@ -1,18 +1,26 @@
 {
+  "extends": "./tsconfig.paths.json",
   "compilerOptions": {
+    "lib": ["esnext"],
     "module": "commonjs",
+    "moduleResolution": "node",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "target": "es6",
+    "target": "ES2020",
```

Comment:
Probably not an issue as we're not doing anything too exotic but it feels weird that we're targeting ES2020 on Node 14